tests/nvme_test.py

```diff
@@ -324,6 +324,14 @@ def _nvme_disconnect(self, subnqn, ignore_errors=False):
         if not ignore_errors and (ret != 0 or 'disconnected 0 ' in out):
             raise RuntimeError("Error disconnecting the '%s' subsystem NQN: '%s %s'" % (subnqn, out, err))
 
+    def _get_sysconf_dir(self):
+        try:
+            makefile = read_file(os.path.join(os.environ['LIBBLOCKDEV_PROJ_DIR'], 'Makefile'))
+            r = re.search(r'sysconfdir = (.*)', makefile)
```

Comment:
I don't have a better idea how to check for *sysconfigdir* here, so I'm just going to pretend I didn't see this.

Reply:
The other way would be to generate some _config.py_ file that would include such string. Not much easier.